### DISCUSSION TITLE
Used for Maven asciidoc generated site

### DISCUSSION THREAD

**Discussion by wimdeblauwe** [peaceiris/actions-gh-pages#589] [Show and tell]
Hi,

Just wanted to take this opportunity to express my gratitude for the work on this library! It made publishing my documentation site a breeze. See https://wimdeblauwe.github.io/error-handling-spring-boot-starter/ if you want to take a look!

regards,

Wim Deblauwe - https://www.wimdeblauwe.com/

**Comment by peaceiris** [comment]
Awesome! Why don't you put the documentation link to the repository website URL?

**Reply by wimdeblauwe** [reply]
Thanks for the tip, did not know about that!
